students/1/main.js

```diff
@@ -31,9 +31,27 @@ const ex6 = () => {
     console.log("Number of Strings: " + result.numStrings)
 }
 
+const ex7 = () => {
+    let person1 = { name: "Alice", age: 22 }
+    let person2 = { name: "Bob", age: 21 }
+    let person3 = { name: "Charlie", age: 23 }
+}
+
 //
 // Your functions here...
 //
+
+let everyone =[
```

Comment:
Let's move this stuff into the ex7 function.

Reply:
Is that right?

Reply:
Let's put all this stuff in the `ex7` function.